### PR TITLE
Added NOT_COMPLETED Result enum member in Shell proto

### PR DESCRIPTION
**Pull request by irsdkv**
Due to https://github.com/mavlink/MAVSDK/pull/944

### CODE REVIEW

File: protos/shell/shell.proto [hunk 1]
```diff
@@ -35,7 +35,8 @@ message ShellResult {
         NO_SYSTEM = 2; // No system is connected
         CONNECTION_ERROR = 3; // Connection error
         NO_RESPONSE = 4; // Response was not received
-        BUSY = 5; // Shell busy (transfer in progress)
+        NOT_COMPLETED = 5; // Response transfer potentially not completed (termination symbol not received)
```
Comment (JonasVautherin) [inline protos/shell/shell.proto]: Wait, now I'm confused :sweat_smile:. What's the difference between BUSY and NOT_COMPLETED? As a user, do I ever want to differentiate them?

Comment (irsdkv) [reply protos/shell/shell.proto]: Hmm. Yes, this really can confuse.
- `BUSY` - shell is busy now
- `NOT_COMPLETED` - in response of current request was not received `ESC`, so we suppose that transfer was interrupted .

May be need to use more clear name (or may be refuse this enum value)?

Comment (JonasVautherin) [reply protos/shell/shell.proto]: I think that it is the same thing for a user: the request has not completed, so a new request cannot be sent. I can't think of a case where a user says "if it's BUSY, do this, and if it's NOT_COMPLETED, do that". If there is no use-case for splitting them, I would vote for reusing `BUSY` (if you guys agree, of course).

Comment (irsdkv) [reply protos/shell/shell.proto]: Hmm. I think this is not the same. But now I think that `NOT_FINISHED` will be more likely to mislead the user than to help. 
May be only add some explaining to the `**brief**`? To notice user to try to increase delay time if response looks incomplete

Comment (JonasVautherin) [reply protos/shell/shell.proto]: ok :slightly_smiling_face:

**Comment by irsdkv** [comment]
Hm. I thought we gave it up :smile:

**Comment by JonasVautherin** [comment]
Not sure I understand that? You were saying that we need to have both `BUSY` and `NOT_COMPLETED`, because the user has to behave differently depending on which one is triggering, right?

**Comment by irsdkv** [comment]
No, sorry for my English :)
I mean that `NOT_COMPLETED` is a redundant case. And talk about leave `BUSY` only and add notice to brief of [shell_command](https://github.com/mavlink/MAVSDK/blob/9895528bf81f687526d1360f06affa2d77e27938/src/plugins/shell/include/plugins/shell/shell.h#L78)

**Comment by JonasVautherin** [comment]
Oh, my mistake, then. If you agree on keeping only `BUSY`, then I'm all for it. Sorry for merging too fast :sweat_smile:.

**Comment by irsdkv** [comment]
Okay, I will reset corresponding commits in  MAVSDK PR and close MAVSDK-Python PR :)

**Comment by JonasVautherin** [comment]
So we should revert this one as well, correct?

**Comment by irsdkv** [comment]
Yes, it is superfluous now

**Comment by JonasVautherin** [comment]
Ok I reverted it. Sorry for the confusion :sweat_smile:

**Pull request merged**